backend/jvm/src/main/scala/zio/logging/backend/NewZIOLogging.scala

```diff
@@ -0,0 +1,19 @@
+package zio.logging.backend
```

Comment:
Probably we need an "examples" project that contains this file.

Reply:
It will part of docs